### DISCUSSION TITLE
Is this project still in development?

### DISCUSSION THREAD

**Discussion by CR09UK4N4** [legendary-gl/legendary#670] [Q&A] [answered]
I heard about this project a few hours ago and need to know if it are still in progress or if it lost

**Comment by hawkeye116477** [comment]
More work is done at https://github.com/Heroic-Games-Launcher/legendary/commits/master/ or at least more releases. I think derrod haven't time for now, but it's rather not abandoned.

**Comment by derrod** [comment, marked as answer]
I have a real job now and not as much interest in working on Epic related stuff, so progress is slow, but it's not abandoned.
